workflow/scripts/osemosys_global/powerplant/user_defined_capacity.py

```diff
@@ -0,0 +1,255 @@
+"""Function to integrate user defined capacities."""
+
+import pandas as pd
+import itertools
+
+from data import get_years
+
+def set_user_defined_capacity(tech_capacity, op_life_dict, df_tech_set, 
+                      df_min_cap_invest, df_max_cap_invest, df_res_cap,
+                      op_life_base, cap_act_base, cap_cost_base, df_iar_final,
+                      df_oar_final, fuel_set, start_year, end_year, region_name, 
+                      DF_IAR_CUSTOM_VAL, DF_OAR_CUSTOM_VAL
```

Comment:
Shouldn't custom capacity adopt the activity ratios of the existing technologies? I get wanting to hard-code in a standard loss for transmission lines, but we should probably use existing or default values for power plants?

Reply:
This has NOT been addressed with the fuel name fix commit!

Reply:
I agree, but this is how it was setup in the legacy code. Personally I think this should be defined by users themselves with an efficiency entry in `user_defined_capacity` and `user_defined_capacity_trn` as mentioned in #173. I'll get to this this or next week.